src/sage/groups/finitely_presented.py

```diff
@@ -1548,22 +1522,21 @@ def sorted_presentation(self):
         L1 = []
         for rel in L0:
             C = [rel]
-            for j in range(len(rel) - 1):
-                C.append(rel[j + 1:] + rel[:j + 1])
+            C.extend(rel[j + 1:] + rel[:j + 1] for j in range(len(rel) - 1))
             C1 = [tuple(-j for j in reversed(l)) for l in C]
             C += C1
             C.sort()
             L1.append(C[0])
         L1.sort()
-        return F/L1
+        return F / L1
 
     def epimorphisms(self, H):
         r"""
         Return the epimorphisms from ``self`` to `H`, up to automorphism of `H`.
 
         INPUT:
 
-        - `H` -- Another group
+        - `H` -- another group
```

Comment:
```suggestion
        - ``H`` -- another group
```
(in the INPUT list, we always use double quotes)